rmw_connext_cpp/package.xml

```diff
@@ -17,15 +17,13 @@
   <build_depend>rcutils</build_depend>
   <build_depend>rmw</build_depend>
   <build_depend>rmw_connext_shared_cpp</build_depend>
-  <build_depend>rosidl_generator_c</build_depend>
-  <build_depend>rosidl_generator_cpp</build_depend>
+  <build_depend>rosidl_runtime_cpp</build_depend>
   <build_depend>rosidl_generator_dds_idl</build_depend>
   <build_depend>rosidl_typesupport_connext_c</build_depend>
   <build_depend>rosidl_typesupport_connext_cpp</build_depend>
   <build_depend>rti-connext-dds-5.3.1</build_depend>
 
   <build_export_depend>connext_cmake_module</build_export_depend>
-  <build_export_depend>rosidl_generator_c</build_export_depend>
   <build_export_depend>rosidl_generator_cpp</build_export_depend>
```

Comment:
Why is this still depending on the generator package?

Reply:
https://github.com/ros2/rmw_connext/pull/399/commits/84893bc4368133086b93054e7048c896e74bb614